<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="usernamelabel2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordlabel2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="repasslabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="repassEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="GendergroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="MaleradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FemaleradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DOBgroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Date Of Birth</string>
   </property>
   <widget class="QComboBox" name="MonthcomboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sep</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oct</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dec</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="YearlineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="DaylineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountTypegroupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="UserradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FGgroupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActioncheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedycheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RomancecheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramacheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HorrorcheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OthercheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RegisterpushButton2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorUsername">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Already existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorRepass">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorAge">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorFields">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>630</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
